lib/ast/AST.cpp

```diff
@@ -1032,6 +1036,14 @@ KOREPattern *KOREAxiomDeclaration::getRequires() const {
               }
             }
           }
+        } else if (equals->getConstructor()->getName() == "\\and" && equals->getArguments().size() == 2) {
```

Comment:
Why is the variable called "equals" if it can be an "and"?

Reply:
historical reasons. I will rename it.